keep/functions/cyaml.py

```diff
@@ -0,0 +1,14 @@
+import yaml
+from yaml import *
+
+
+def safe_load(stream):
+    return yaml.load(stream, Loader=yaml.CSafeLoader)
+
+def dump(data, stream=None, Dumper=None, **kwds):
+    Dumper = Dumper or yaml.CDumper
+    return yaml.dump(data, stream, Dumper=Dumper, **kwds)
+
+def add_representer(data_type, representer, Dumper=None):
+    Dumper = Dumper or yaml.CDumper
+    Dumper.add_representer(data_type, representer)
```

Comment:
nice! why isn't this the goto for the builtin yaml library?

Reply:
have no idea, why they didn't provide usable API for c-loaders

Reply:
🆗 lets hope this doesn't backfire us